.travis.yml

```diff
@@ -0,0 +1,11 @@
+# build documentation in `source` branch and commit result to `master`, from
+# https://github.com/matplotlib/matplotlib/blob/c259a556/.travis.yml#L147-L169
+
+sudo: false
+
+language: python
+
+script:
+  - make.bat html
```

Comment:
Yes. Sorry, I am on Windows so I forgot that it is for Travis and not for AppVeyor.